testproject/Assets/ManualTests/Scripts/NetworkPrefabPool.cs

```diff
@@ -0,0 +1,360 @@
+                }
+                else
+                {
+                    NetworkManager.PrefabHandler.AddHandler(ServerObjectToPool, m_MyCustomPrefabSpawnHandler);
+                }
+            }
+            else if (!IsServer)
+            {
+                Debug.LogWarning($"Failed to register custom spawn handler and {nameof(EnableHandler)} is set to true!");
+            }
+        }
+    }
+
+    /// <summary>
+    /// When disabled, stop spawning objects
+    /// </summary>
+    private void OnDisable()
+    {
+        m_IsSpawningObjects = false;
+        if (NetworkManager.Singleton && EnableHandler && m_MyCustomPrefabSpawnHandler != null)
+        {
+            var no = ServerObjectToPool.GetComponent<NetworkObject>();
+            NetworkManager.Singleton.PrefabHandler.RemoveHandler(no);
+            m_MyCustomPrefabSpawnHandler = null;
+        }
+    }
+
+    /// <summary>
+    /// General clean up
+    /// The custom prefab handler is de-registered here
+    /// </summary>
+    private void OnDestroy()
+    {
+        if (SwitchScene)
+        {
+            SwitchScene.OnSceneSwitchBegin -= OnSceneSwitchBegin;
+        }
+    }
+
+    // Start is called before the first frame update
+    private void Start()
+    {
+        if (SpawnSliderValueText != null)
+        {
+            SpawnSliderValueText.text = SpawnsPerSecond.ToString();
+        }
+
+        if (SwitchScene)
+        {
+            SwitchScene.OnSceneSwitchBegin += OnSceneSwitchBegin;
+        }
+
+        //Call this again in case we didn't have access to the NetworkManager already (i.e. first scene loaded)
+        RegisterCustomPrefabHandler();
+
+    }
+
+    #endregion
+
+    /// <summary>
+    /// Detect when we are switching scenes in order
+    /// to assure we stop spawning objects
+    /// </summary>
+    private void OnSceneSwitchBegin()
+    {
+        if (IsServer)
+        {
+            StopCoroutine(SpawnObjects());
+
+            if (m_ObjectPool != null)
+            {
+                foreach (var obj in m_ObjectPool)
+                {
+                    var networkObject = obj.GetComponent<NetworkObject>();
+                    if (networkObject.IsSpawned)
+                    {
+                        networkObject.Despawn();
+                    }
+                    Destroy(obj);
+                }
+                m_ObjectPool.Clear();
+            }
+        }
+    }
+
+    /// <summary>
+    /// Override NetworkBehaviour.NetworkStart
+    /// </summary>
+    public override void NetworkStart()
+    {
+        InitializeObjectPool();
+        if (IsServer)
+        {
+            if (isActiveAndEnabled)
+            {
+                m_DelaySpawning = Time.realtimeSinceStartup + InitialSpawnDelay;
+                StartSpawningBoxes();
+            }
+        }
+        base.NetworkStart();
```

Comment:
you don't actually need to call the base method here at all (there's nothing in the base, there never will be).

Reply:
I tend to make that a habit in the event we want some "default thing" to happen... but yeah generally speaking as it is today.. it isn't needed.